.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
```

Comment:
nice touch to have different templates

Reply:
Yeah! You'll get a new intermediate step now when you go to create a ticket that prompts you to pick one of the defined types.